document-store/src/main/java/org/hypertrace/core/documentstore/query/Query.java

```diff
@@ -93,7 +99,7 @@ public static QueryBuilder builder() {
       return Collections.emptyList();
     }
 
-    return aggregation.getExpressions();
+    return Collections.unmodifiableList(aggregation.getExpressions());
```

Comment:
Just thinking when would the clients use these getters which returns unmodifiableList ?

Reply:
The client's wouldn't have a case to use these. These methods would mostly be used by the parsers.